### ISSUE TITLE
Botania shears (and possibly other tools) can't harvest some plants that Vanilla (Apotheosis?) can.

### ISSUE THREAD

**Issue by ThePB** [NillerMedDild/Enigmatica6#3239]
### Modpack Version

0.5.11

### Describe your issue

As title. I really have no knowledge of other tools that could be used to imitate shears to try to use them, too, but i replicated (log attached) something that happened a bit ago and i simply thought of "Oh well those plant might simply not be harvestable".

I don't have a full list, but the "Oh the biomes you'll go" Imparius Bushes and Oddity (? the yellow and blue ones from the end desert) bushes are among the plants that have different behavior with the set of tools.

I also didn't try other tools (again, not much knowledge of what else to use to begin with) so i don't know which mod is "not behaving".

[latest bushes.log](https://github.com/NillerMedDild/Enigmatica6/files/7162503/latest.bushes.log)

### Crash Report

N/A

### Latest Log

Above

### Have you modified the modpack?

Yes

### User Modifications

Tetra spear-> swords for enchantment! Waystones/JM compat config! Number of Backups! Nothing that i think matters.

### Did the issue happen in singleplayer or on a server?

Singleplayer

**Comment by NillerMedDild** [comment]
Thanks for the report. I found the issue and have made a pull request fixing it: https://github.com/AOCAWOL/BYG-1/pull/35

**Comment by ThePB** [comment]
Sorry to write here again , but i suspect that the exact same thing might be happening with "Project: Vibrant Journeys" too. Their short grass isn't harvested with Botania shears, but it does harvest with normal (Apo.) ones. Not that it is useful in any way by itself, but.... I guess this info doesn't warrant opening a new issue...

**Comment by NillerMedDild** [comment]
I agree, commenting here was the right approach 👍 I'll get those fixed too, please feel free to comment with any other non-shearables you find. I'd prefer a separate comment for each finding, that way I'm sure to be notified :)

**Comment by NillerMedDild** [comment]
PRs for PVJ and Environmental:
https://github.com/OrderedChaos-Dev/ProjectVibrantJourneys/pull/164
https://github.com/team-abnormals/environmental/pull/242

**Comment by NillerMedDild** [comment]
Fixed in PVJ \o/

**Comment by NielsPilgaard** [comment]
Mostly fixed I think 😄
